ldes-fragmentisers/ldes-fragmentisers-timebased-hierarchical/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/timebasedhierarchical/services/TimeBasedRelationsAttributer.java

```diff
@@ -29,11 +29,16 @@ public TimeBasedRelationsAttributer(FragmentRepository fragmentRepository,
 
 	public void addInBetweenRelation(Fragment parentFragment, Fragment childFragment) {
 		FragmentationTimestamp timestamp = timestampFromFragmentPairs(childFragment);
-		TreeRelation parentChildRelation = new TreeRelation(config.getFragmentationPath(),
+		TreeRelation parentGteRelation = new TreeRelation(config.getFragmentationPath(),
 				childFragment.getFragmentId(),
-				timestamp.asString(), timestamp.getType(),
```

Comment:
I don't think we still need the method `getType` on  FragmentationTimestamp, we also won't need it on granularity anymore and we won't need the field `type` on Granularity anymore either. Can you remove all of these so that we don't have dangling/unused code?